autorest/azure/environments.go

```diff
@@ -3,12 +3,24 @@ package azure
 import (
 	"fmt"
 	"net/url"
+	"strings"
 )
 
 const (
 	activeDirectoryAPIVersion = "1.0"
 )
 
+var environments map[string]Environment
+
+func init() {
```

Comment:
techically, no need to have`init()` you can just assign value in `var` declaration above.